File: src/p1288/Main.cpp
#include<iostream>
#include<cstdio>

#include<vector>
#include<string>
#include<algorithm>

using namespace std;

int main()
{
	 	freopen("test.txt","r",stdin);
	int cases;
	scanf("%d",&cases);
	while(cases-->0)
	{
		string a,b;
		int k;
		cin>>a>>b>>k;
		vector<int> diff(a.length());
		int j;
		for(j=0;j<a.length();j++)
		{
			diff[j]=(a[j]-b[j]>=0)?(a[j]-b[j]):(b[j]-a[j]);
		}
		sort(diff.begin(),diff.end());
		for(j=a.length()-k;j<a.length();j++)
		{
			if(diff[j]==0) 
				diff[j]=1;
			else 
				diff[j]=0;
		}
		int sum=0;
		for(j=0;j<diff.size();j++) sum+=diff[j];

		printf("%d\n",sum);
	}
	return 0;
}

File: src/p1288/Main.java
package p1288;

import java.util.Arrays;
import java.util.Scanner;

/**
 * Copyright(C) 2015,Cao Qingqing , Wuhan University All rights reserved.
 * 
 * This class is submitted for solving the problem 1288 in WOJ
 * 
 * @author : Qingqing Cao, Computer School, Wuhan University
 * 
 */
public class Main {
	public static void main(String args[]) {

		Scanner input = new Scanner(System.in);
		int cases = input.nextInt();
		char[] a, b;
		int k;
		int[] diff;
		while (cases-- > 0) {
			a = input.next().toCharArray();
			b = input.next().toCharArray();
			k = input.nextInt();
			diff = new int[a.length];
			for (int i = 0; i < a.length; i++) {
				diff[i] = Math.abs(a[i] - b[i]);
			}
			Arrays.sort(diff);
			int sum = 0;
			for (int i = diff.length - k; i < diff.length; i++) {
				if (diff[i] == 0)
					diff[i] = 1;
				else
					diff[i] = 0;
			}
			for (int i = 0; i < diff.length; i++) {
				sum += diff[i];
			}
			System.out.println(sum);

		}
		input.close();
	}
}
